### PR TITLE
Views

### PR DESCRIPTION
**Pull request by ridaarif98**
- Implement the design from the sneak peek wireframes.
- Make sure that every view is displaying data.
- Run `rake db:seed` to populate data in the database

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,23 @@
...
+            <h4>Post #<%=i+1%></h4>
+            <p><%= post.text%></p>
+            <p class="show-li">Coments : <%= post.comments_counter.to_i%>  Likes : <%= post.likes_counter.to_i %> </p>
+
+        <% end %>
+        </li>
+        <div class="<%= comments?(post)%> user-comments">
+            <% post.comments.each do |comment|%>
+                <p>
+                    <%=comment.user.name%> :
```
Comment (SajjadAhmad14) [inline app/views/posts/index.html.erb]: Please use `post.recent_comments` to render 5 most recent comments in the app/views/posts/index.html.erb page.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,23 @@
-<h1>Here is the list of all posts of a user</h1>
```
Comment (SajjadAhmad14) [inline app/views/posts/index.html.erb:1]: Please add `rails db: create, rails db:migrate, and rails db:seed` and `yarn install` commands in the installation instructions of README.

File: db/migrate/20211206132802_rename_author_column_in_comments.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class RenameAuthorColumnInComments < ActiveRecord::Migration[6.1]
+    def change
+      rename_column :comments, :author_id, :user_id
+    end
```
Comment (SajjadAhmad14) [inline db/migrate/20211206132802_rename_author_column_in_comments.rb]: Please notice the error when running r `rails db:migrate` and fix it so the user can install the project without any errors.
migrate

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Rida, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @ridaarif98, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ridaarif98** [comment]
@Dipeshtwis  thank you

**Pull request merged**